docs/administrative-requirements.md

```diff
@@ -0,0 +1,136 @@
+
+All VA systems are given an entry with a unique number in the eMASS system. The Enterprise Event Bus eMASS ID is 2350. 
+
+Getting an eMASS ID is a prerequisite for [System Categorization](#fisma-system-categorization). 
+
+**For non-LHDI clients**:
+
+* If a prospective client system does not have an eMASS ID, it will need to submit a [LEAF intake request](https://leaf.va.gov/NATIONAL/103/cybersecurity_request_portal/) to get one.
+
+**For new LHDI clients**:
+
+* For new teams on Lighthouse Delivery Infrastructure (LHDI), please consult your LHDI Enablement Liaison to start the LEAF intake request process and LHDI-specific onboarding. 
+
+## Authority to Operate (ATO)
+
+ATO, or “Authority to Operate,” indicates that your system has been evaluated by VA and given permission to deploy code in production.
+
+Teams that have an active ATO should review it to understand the implications of integrating with other systems.
+
+**For non-LHDI clients**:
+
+ATO is evaluated once eMASS system registration has been completed and approved. Teams that do not have an active ATO should begin the process as early as possible in the Enterprise Event Bus integration process.
+
+**For new LHDI clients**:
+
+For new teams within the Lighthouse Delivery Infrastructure, please consult your LHDI Enablement Liaison to begin the ATO process.
+
+## FISMA - System Categorization
+
+As part of the eMASS process, System Categorization will evaluate the impact to the organization of loss or compromise to the data in the application. The outcome of the System Categorization process is a FISMA risk level rating of Low, Moderate, or High.
+
+If a prospective client system has not been through System Categorization, it will need to follow the steps outlined on this [GRC System Categorization page](https://confluence.devops.va.gov/display/VAExternal/GRC+-+System+Categorization). Note that completing a Privacy Threshold Analysis (PTA) document is a prerequisite for System Categorization.
+
+**Note**: For teams handling ePHI (medical information specific to an individually identifiable patient), the HIPAA Security Rule applies and further review will be required. Support for this can be requested by sending an email to [VHAHCSDevelopmentSecurity2@va.gov](mailto:VHAHCSDevelopmentSecurity2@va.gov).
+
+## PTA and PIA
+
+A PTA, or Privacy Threshold Analysis, is a required document used to determine if an IT system, program, project, or boundary is privacy-sensitive and requires additional privacy compliance documentation such as a PIA or SORN.  It is also the first step of the privacy compliance documentation process. 
+
+A PIA, or Privacy Impact Assessment, is a public document that describes:  
+
+* What PII VA is collecting  
```

Comment:
```suggestion
* What PII the system is collecting  
```

Reply:
I'll accept this, but this was taken directly from VA's PIA docs.